test/msw-api/setup-server/printHandlers.test.ts

```diff
@@ -36,6 +36,7 @@ afterAll(() => {
 test('lists all current request handlers', () => {
   server.printHandlers()
 
+  // Test failed here, commenting so it shows up in the PR
   expect(console.log).toBeCalledTimes(6)
 
   expect(console.log).toBeCalledWith(`\
```

Comment:
I am struggling with these tests and can't figure out why!! I wonder if you have had this problem before and can give me some pointers on this.  😔

(It does seem like it's outputting the correct number of calls, however, the test results were printed in a batch instead of in sequence to the terminal)

Reply:
I've fixed the tests, the CI is passing now.